### ISSUE TITLE
remix.init failed

### ISSUE THREAD

**Issue by djake** [remix-run/indie-stack#140]
### What version of Remix are you using?

1.6.8

### Steps to Reproduce

Hello - First time trying out Remix and the [Getting Started](https://remix.run/docs/en/v1/tutorials/blog) tutorial is immediately failing for me.

macOS Monteray
node version 18.7.0
npm 8.15.0

```sh
npx create-remix --template remix-run/indie-stack jake-start
? TypeScript or JavaScript? TypeScript
? Do you want me to run `npm install`? Yes
```

This is a new machine and I assume you would be aware of a problem like this so may it will end up being a me thing, but everything else seems to be working on my end. 

### Expected Behavior

No CLI errors and "the app should be running," including a 200 at `localhost:3000`.

### Actual Behavior

```sh
🚨 Oops, remix.init failed

Cannot read properties of undefined (reading 'length')
```

More:

```sh
jake:~/repos/remix-getting-started$ npx create-remix --template remix-run/indie-stack jake-start
? TypeScript or JavaScript? TypeScript
? Do you want me to run `npm install`? Yes
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated

added 1312 packages, and audited 1313 packages in 55s

231 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
💿 Running remix.init script
🚨 Oops, remix.init failed

Cannot read properties of undefined (reading 'length')
```

There ends up being something which looks like an app in the folder, and `npm run dev` initially starts a server, but then loading a page results in:

```
Error: error: Environment variable not found: DATABASE_URL.
  -->  schema.prisma:3
   |
 2 |   provider = "sqlite"
 3 |   url      = env("DATABASE_URL")
```

**Comment by MichaelDeBoey** [comment]
This will be fixed by #138

**Comment by Zakini** [comment]
What's the workaround in the meantime? Should we use an older version, and if so, how do we do that with `npx remix-create` and what's the latest version that doesn't have this bug?

**Comment by TimonVS** [comment]
@Zakini I ran into the same problem, I've outlined the steps below to run the fork:

1. Check out the fork:

```sh
git clone https://github.com/MichaelDeBoey/indie-stack.git
```

2. Switch to the branch containing the fixes: [`fix-convert-to-js`](https://github.com/MichaelDeBoey/indie-stack/tree/fix-convert-to-js)

```sh
git checkout -b fix-convert-to-js origin/fix-convert-to-js
```

3. Install dependencies.

```sh
npm install
```

4. Run init script as described in [README](https://github.com/remix-run/indie-stack#development).

```sh
npx remix init
```

5. Test if everything works by running the app.

```sh
npm run dev
```

6. Remove `.git` directory to remove references to @MichaelDeBoey's repository.

```sh
rm -rf .git
```

**Comment by Zakini** [comment]
Ah, if using a specific branch from a fork is the answer, you do that more easily with `npx create-remix`:

```
npx create-remix --template https://github.com/MichaelDeBoey/indie-stack/tree/fix-convert-to-js
```

**Comment by MichaelDeBoey** [comment]
As @Zakini said: `npx create-remix --template https://github.com/MichaelDeBoey/indie-stack/tree/fix-convert-to-js` will indeed do the trick

**Comment by MattDClarke** [comment]
I created a project using `npx create-remix --template https://github.com/MichaelDeBoey/indie-stack/tree/fix-convert-to-js`. I get the following warnings and errors in my console:

- `react-dom.development.js:86 Warning: Expected server HTML to contain a matching <head> in <html`

- `Uncaught Error: Hydration failed because the initial UI does not match what was rendered on the server.`

- `react-dom.development.js:86 Warning: An error occurred during hydration. The server HTML was replaced with client content in <#document>.`

- `Uncaught Error: There was an error while hydrating. Because the error happened outside of a Suspense boundary, the entire root will switch to client rendering.`



Also, I'm going through the [Developer Blog](https://remix.run/docs/en/v1/tutorials/blog) tutorial, the action does not return the form data:

```react
export const action: ActionFunction = async ({ request }) => {
    const formData = await request.formData();
    console.log({formData});
    return redirect("/posts/admin");
} 
```



This logs:

```bash
{ formData: FormData {} }
```

Windows 10
Node v16.13.1
npm v8.1.2
